### PR TITLE
Update documentation for aci_interface_policy_cdp

### PR DESCRIPTION
**Pull request by kbreit**
##### SUMMARY
Documentation only changes. Improved the examples and changed the `version_added` from 2.8 to 2.9.

##### ISSUE TYPE
- Docs Pull Request

+label: docsite_pr
##### COMPONENT NAME
aci_interface_policy_cdp

### CODE REVIEW

File: lib/ansible/modules/network/aci/aci_interface_policy_cdp.py [hunk 1]
```diff
@@ -52,16 +52,30 @@
...
-    username: '{{ username }}'
-    password: '{{ password }}'
-    cdp_policy: '{{ cdp_policy }}'
-    description: '{{ description }}'
-    admin_state: '{{ admin_state }}'
-  delegate_to: localhost
+- name: Create CDP Test Policy
+  aci_interface_policy_cdp:
+    name: Ansible_CDP_Test_Policy
+    host: apic.acmecorp.com
```
Comment (samccann) [inline lib/ansible/modules/network/aci/aci_interface_policy_cdp.py]:
```suggestion
    host: apic.example.com
```

File: lib/ansible/modules/network/aci/aci_interface_policy_cdp.py [hunk 2]
```diff
@@ -52,16 +52,30 @@
...
+    username: admin
+    password: adminpass
+    state: present
+
+- name: Remove CDP Test Policy
+  aci_interface_policy_cdp:
+    name: Ansible_CDP_Test_Policy
+    host: apic.acmecorp.com
```
Comment (samccann) [inline lib/ansible/modules/network/aci/aci_interface_policy_cdp.py]:
```suggestion
    host: apic.example.com
```

File: lib/ansible/modules/network/aci/aci_interface_policy_cdp.py [hunk 3]
```diff
@@ -52,16 +52,30 @@
...
+	  username: admin
+	  password: adminpass
+	  output_level: debug
+	  state: absent
+
+- name: Query CDP Policy
+  aci_interface_policy_cdp:
+    host: apic.acmecorp.com
```
Comment (samccann) [inline lib/ansible/modules/network/aci/aci_interface_policy_cdp.py]:
```suggestion
    host: apic.example.com
```

**Review by samccann — CHANGES_REQUESTED** [review]

**Comment by kbreit** [comment]
@samccann Updated.

**Review by samccann — APPROVED** [review]

**Comment by acozine** [comment]
This module did not actually get into the 2.8 release, but it was merged with `version_added` set to `2.8`. Shippable is now failing only on the `version_added` mismatch. Merging on a test failure to get the correct `version_added` metadata along with other improvements. 

Thanks @kbreit.

**Pull request merged**